### DISCUSSION TITLE
recommended buffer type to use in fluentd

### DISCUSSION THREAD

**Discussion by vishalk663** [fluent/fluentd#4324] [General]
We have around 100 million documents inserted into the Elasticsearch cluster daily. Elasticsearch cluster are configured in Kubernetes with fluentd. 

So which buffer type we should use memory or file buffer and why. 

what will be the recommended configuration of it? 

we are not considering speed here but data should not be loss in elasticsearch cluster or fluentd should not failed to flush the logs into elasticsearch cluster.

Thanks

**Comment by kenhys** [comment]
duplicate of #4325
